Detectors/TRD/trap/trap_tpp.cpp

```diff
@@ -0,0 +1,575 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "trap_tpp.h"
+#include <stdio.h>
+#include <math.h>
+
+// $Id$:
```

Comment:
Is this a new file? Consider stating where it comes from (if applicable).

Reply:
as comment directly above.